logging/log_collection_forwarding/log-forwarding.adoc

```diff
@@ -13,14 +13,18 @@ Administrators can create `ClusterLogForwarder` resources that specify which log
 
 Administrators can also authorize RBAC permissions that define which service accounts and users can access and forward which types of logs.
 
-////
 include::modules/log-forwarding-modes.adoc[leveloffset=+1]
 
 [id="log-forwarding-enabling-multi-clf-mode"]
 == Enabling multi log forwarder mode for a cluster
 
 To use multi log forwarder mode, you must create a service account and cluster role bindings for that service account. You can then reference the service account in the `ClusterLogForwarder` resource to control access permissions.
 
+[IMPORTANT]
+====
+If you want to use multi log forwarder mode in namespaces other than the `openshift-logging` namespace, you must xref:../../logging/cluster-logging-upgrading.adoc#logging-operator-upgrading-all-ns_cluster-logging-upgrading[update the Cluster Logging Operator to watch all namespaces]. This functionality is supported by default in new Cluster Logging Operator version 5.8 installations.
```

Comment:
..multi-log forwarder feature....

Reply:
Why would it be named differently here vs other places in the doc?

Reply:
I don't recall the other places but the feature is "multi log forwarder" and that is what is being delivered.  To fully enable it in all namespaces action needs to be taken.  I suggest not using "mode" because that is a way to try and convey to admins how it used to work and what they are familiar with (e.g. legacy) versus how the operator will function going forward (e.g. multi).  The issue mostly with how I wrote the enhancement is that "mode" always strikes me as an intentional switch/know/config that is done.  Admins really don't do anything in that regards other then redeploy.

I'm probably not hard pressed either way.  If as an outside observer this conveys what it needs to then I'm OK with that

Reply:
Updated across the board to "implementation" or "feature" instead of "mode". Take a look and lmk what you think 🙂